conventions/documentation/quick-starts.md

```diff
@@ -0,0 +1,205 @@
+5. **Badges:** Quick start tiles can have a number of badges. Each quick start should have the time estimate in a badge. If a quick start has been started or completed, show the relevant badge: In progress or Completed.
+6. **Prerequisites (Optional):** When applicable, detail what prerequisites users / clusters have to meet before completing the quick start. The prerequisites should appear underneath the description.
+
+### Overview / details
+The first screen of any quick start should give an overview of the quick start. The overview can include information about the benefits of completing the quick start (e.g., why you should use Serverless) and / or a summary of the tasks to be completed.
+![details](../images/qs-details.png)
+
+### Task screens
+Each quick start will walk a user through a set number of tasks.
+![task](../images/qs-task.png)
+1. **Numbered tasks:** Each quick start should consist of a set of high-level tasks. These should represent the overarching steps that users will need to complete in order to successfully finish the quick start. It is recommended that each quick start have at least two tasks, but no more than five or six. Next to the task header that is currently open / active, show the task number out of the total number of tasks. Example: 2 of 3.
+2. **Substeps:** Substeps provide the specific, granular details of how users can complete each numbered task. UI elements should be formatted to match the guidelines below.
+3. **Check your work:** After clicking Next on a task screen, users are presented with a Check your work feature. Each task should have a validation question to present for the Check your work feature. Users can choose to skip this step by clicking the Next button again.
+4. **Context (optional):** When needed, tasks and substeps should provide context to the user. Be clear as to why the user is taking the actions, how it helps them, etc. Don’t be afraid to add content that isn’t strictly instructional!
+
+### Additional components
+The following are additional quick start components.
+1. **Metadata:** Quick starts should have metadata that can be used for sorting and filtering. This can include product names, keywords, users, etc.
+2. **Version, timestamp:** If quick starts are only compatible with certain versions of OpenShift, this should be noted on the quick start tile, in the quick start description, and in the quick start instructions.
+3. **RBAC specs:** Quick starts should be appropriately flagged to determine which perspectives they may be accessible from.
+4. **Next quick start (Optional):** When applicable, upon completion of a quick start, surface a link to the next quick start in the “bundle”.
+
+## (WIP) Part III: How do I write quick start content?
+This section is a work in progress. Keep an eye out for a more detailed style guide!
+
+### How to write a task
```

Comment:
Should we also add "Which persona am I writing this for?" to ensure they are in the right perspective?